ocpi-tariffs/src/types/number.rs

```diff
@@ -84,7 +84,7 @@ impl Sub for Number {
     type Output = Self;
 
     fn sub(self, rhs: Self) -> Self::Output {
-        Self(self.0 / rhs.0)
+        Self(self.0 - rhs.0)
```

Comment:
I would consider using `saturating_sub` and the other safe variants for all arithmetic operations.
At the moment you wrapping will occur silently.